editor/editor_themes.cpp

```diff
@@ -1115,9 +1120,15 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_stylebox("camera", "GraphEditMinimap", style_minimap_camera);
 	theme->set_stylebox("node", "GraphEditMinimap", style_minimap_node);
 
-	Ref<Texture2D> resizer_icon = theme->get_icon("GuiResizer", "EditorIcons");
-	theme->set_icon("resizer", "GraphEditMinimap", flip_icon(resizer_icon, true, true));
-	theme->set_color("resizer_color", "GraphEditMinimap", Color(1, 1, 1, 0.65));
+	Ref<Texture2D> minimap_resizer_icon = theme->get_icon("GuiResizer", "EditorIcons");
+	Color minimap_resizer_color;
+	if (dark_theme) {
+		minimap_resizer_color = Color(1, 1, 1, 0.65);
+	} else {
+		minimap_resizer_color = Color(0, 0, 0, 0.65);
+	}
+	theme->set_icon("resizer", "GraphEditMinimap", flip_icon(minimap_resizer_icon, true, true));
+	theme->set_color("resizer_color", "GraphEditMinimap", minimap_resizer_color);
```

Comment:
I would suggest pushing theme element overriding into `notification`/`NOTIFICATION_THEME_CHANGED`. Because currently, it remains light on the light theme if the user changes the theme:
![image](https://user-images.githubusercontent.com/3036176/105740949-c9099880-5f4a-11eb-9c87-ec03f2e3b791.png)

Reply:
It updates correctly for me. This is similar to how we handle all other color properties in the theme.


https://user-images.githubusercontent.com/11782833/105743131-65349f00-5f4d-11eb-8703-202baa47f7fd.mp4

Reply:
Ah, sorry, I mixed up the Godot windows :/ Tested, seems ok.